starknet_py/serialization/data_serializers/output_serializer.py

```diff
@@ -0,0 +1,38 @@
+from dataclasses import dataclass, field
+from typing import Dict, Generator, List, Tuple
+
+from starknet_py.serialization import CairoDataSerializer
+from starknet_py.serialization._context import (
+    DeserializationContext,
+    SerializationContext,
+)
+
+
+@dataclass
+class OutputSerializer(CairoDataSerializer[List, Tuple]):
+    """
+    Serializer for function output.
+    Can't serialize anything.
+    Deserializes data to a Tuple.
+
+    Example:
+        [1, 1, 1] => (340282366920938463463374607431768211457)
```

Comment:
This example is unclear to me. Could we add types for the output above, like in `EnumSerializer`?